projects/kit/directives/skeleton/skeleton.style.less

```diff
@@ -0,0 +1,37 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+@keyframes tuiSkeleton {
+    from {
+        opacity: 0.03;
+    }
+
+    to {
+        opacity: 0.06;
+    }
+}
+
+[tuiSkeleton] {
+    &._skeleton {
+        color: transparent;
+        background: var(--tui-base-01) !important;
+        box-shadow: none !important;
+        filter: contrast(0) brightness(0);
+        animation: tuiSkeleton ease-in-out 1s infinite alternate;
+
+        &[data-tui-skeleton] {
+            background: transparent !important;
+
+            &:before {
+                content: attr(data-tui-skeleton);
+                background: var(--tui-base-01);
+                font-size: smaller;
+                box-decoration-break: clone;
+                border-radius: 0.25rem;
+            }
+        }
+    }
+}
+
+[tuiTheme='night'] [tuiSkeleton]._skeleton {
```

Comment:
`dark` ?

https://developer.mozilla.org/ru/docs/Web/CSS/@media/prefers-color-scheme#dark

Reply:
Yes, we need to do it everywhere at once in a separate PR